src/python/WMCore/Storage/DeleteMgr.py

```diff
@@ -210,30 +213,46 @@ def __call__(self, fileToDelete):
             msg = "Unable to delete file:\n"
             msg += fileToDelete['LFN']
             raise StageOutFailure(msg, **fileToDelete)
-
-    def deleteLFN(self, lfn, override=None):
+    
+    def deleteLFN(self, lfn, stageOut_rfc):
         """
         deleteLFN
 
-        Given the lfn and local stage out params, invoke the delete
+        Given the lfn and an stageOut Rucio file catalog pair, invoke the delete
+
+        """
+        from WMCore.Storage.StageOutMgr import searchRFC 
```

Comment:
keep as is to avoid circular import
`File "/home/dmwm/wmcore_unittest/WMCore/test/python/WMCore_t/Storage_t/DeleteMgr_t.py", line 13, in <module>
    from WMCore.Storage.DeleteMgr import DeleteMgr
  File "/home/dmwm/wmcore_unittest/WMCore/src/python/WMCore/Storage/DeleteMgr.py", line 26, in <module>
    from WMCore.Storage.StageOutMgr import searchRFC
  File "/home/dmwm/wmcore_unittest/WMCore/src/python/WMCore/Storage/StageOutMgr.py", line 19, in <module>
    from WMCore.Storage.DeleteMgr import DeleteMgr
ImportError: cannot import name 'DeleteMgr' from partially initialized module 'WMCore.Storage.DeleteMgr' (most likely due to a circular import) (/home/dmwm/wmcore_unittest/WMCore/src/python/WMCore/Storage/DeleteMgr.py)`

Reply:
ok